includes/Story_Post_Type.php

```diff
@@ -397,6 +397,7 @@ protected static function get_allowed_mime_types() {
 				'image/jpeg',
 				'image/jpg',
 				'image/gif',
+				'image/svg',
```

Comment:
Looks like `image/svg+xml` needs to be added here, doesn't seem to work otherwise for uploading in the editor. Or alternatively, the `+xml` part could be removed in the FE when checking for valid type. Thoughts?

Reply:
`image/svg` is not a valid mime type. Only [`image/svg+xml` is](https://www.w3.org/TR/SVG11/mimereg.html).

Reply:
WordPress saves and processes it as `image/svg`. Which is why the original PR didn't work :man_shrugging:

Reply:
![Screenshot from 2020-03-27 17-56-01](https://user-images.githubusercontent.com/237508/77785587-4f67fc00-7054-11ea-9043-d6b5ece312a2.png)
Example output from the REST API.

Reply:
@spacedmonkey The custom FE logic still needs to receive `image/svg+xml` in `allowedMimeTypes`.

Reply:
(It does work for WP uploads just with `image/svg` but not for the uploads within our editor)

Reply:
This enables svg in all contexts.

Reply:
> @spacedmonkey The custom FE logic still needs to receive `image/svg+xml` in `allowedMimeTypes`.

Why?

Reply:
Because the FE implementation for drag-drop isn't related to WP and is using the original mime types, this makes dragging directly to editor fail. Upload to WP library would work, dragging not.

See the `isValidType` check when uploading SVG:
<img width="690" alt="Screenshot 2020-03-30 at 11 20 33" src="https://user-images.githubusercontent.com/3294597/77896921-44da7c00-7279-11ea-8172-c91ce7680f48.png">

The type is `image/svg+xml`, howeger, the `allowedMimeTypes`:
<img width="536" alt="Screenshot 2020-03-30 at 11 24 57" src="https://user-images.githubusercontent.com/3294597/77896961-53289800-7279-11ea-9377-c23c65a9484a.png">

The result:
<img width="643" alt="Screenshot 2020-03-30 at 11 26 59" src="https://user-images.githubusercontent.com/3294597/77897018-6471a480-7279-11ea-8cc8-d05f06dc6e73.png">


This means that for using WordPress media library, we can keep just `image/svg` for filtering the allowed mime types, however, via config to the FE we'd need to pass `image/svg+xml` (for dropping a file directly to the editor).

Reply:
I have added both for good messure in https://github.com/google/web-stories-wp/pull/832/commits/6ffb4f0f7afc2e5d7b13918e664816394b0948cc.